# array of pose reading for all contacts
Contact[] contact_list
